### DISCUSSION TITLE
SVG of logo?

### DISCUSSION THREAD

**Discussion by loganmarchione** [containrrr/watchtower#1513] [General] [answered]
Hello there! 👋

I'm the maintainer of [homelab-svg-assets](https://github.com/loganmarchione/homelab-svg-assets) and would like to add the [watchtower icon](https://github.com/containrrr/watchtower/blob/main/logo.png) to my project. However, I didn't see any brand guidelines or a vector (SVG) version of the icon. Could you please provide one or both of these?

**Comment by simonaronsson** [comment, marked as answer]
No brand guidelines to speak of other than: use your best judgement and don't be an ass :) SVG attached below. 
![watchtower-logo](https://user-images.githubusercontent.com/1596025/209564198-f3050206-714a-4484-9924-6bf071ec7841.svg)

**Reply by loganmarchione** [reply]
Thank you!
